maflib/column_values.py

```diff
@@ -32,6 +32,15 @@ class NullableYOrNEnum(MafEnum):
     Yes = "N"
```

Comment:
Just noticed..... 🙃

Reply:
oh nice find, will fix this before merging

Reply:
Changed this and added in unit tests